__tests__/connectRoutes.js

```diff
@@ -9,6 +9,10 @@ import { NOT_FOUND } from '../src/index'
 import redirect from '../src/action-creators/redirect'
 import pathToAction from '../src/pure-utils/pathToAction'
 
+beforeEach(() => {
+  window.SSRtest = false
```

Comment:
In future in rudy, I think its a good idea to invest time making some isolated unit tests that test only one module at a time. That way we can easily pass in optional arguments like `isServer = () => true` one test at a time instead of worrying about global state.

Reply:
Sure, that's a good idea.

Reply:
yeah a good idea!